docs/reference-guide/network-ports.rst

```diff
@@ -0,0 +1,49 @@
+Network Ports
+=============
+
+etcd Ports
+----------
+
+etcd is a cluster of servers storing keys and values
+distributing a database.
```

Comment:
The keys and values don't distribute a database.

See https://coreos.com/etcd/ for a better explanation.